### ISSUE TITLE
Help reproducing NCT results from paper

### ISSUE THREAD

**Issue by s-ihler** [maxwell0027/PEFAT#4]
Hi there, 
thanks for posting your code online :)

We are currently trying to reproduce the results from your paper for the NCT dataset.
We tried the current hyperparams in your code (batchsize 32, num_labeled_sample 8 and lr 1e-4) and the ones mentioned in your paper (batchsize 64, num_labeled_sample 16 and lr 1e-3). Number of warmup epochs is always 30 and pefat epochs is always 50. We used the data splits in this repo.

We used the transforms from TransformTwice for weak and strong augmentations: 
1st stage (warmup):
```
# # # note that weak and base for warm up and baseline training
out1 = self.weak_transform(inp)
out2 = self.base_transform(inp)
```
2nd stage (pefat): 
```
out1 = self.color_transform(inp)
out2 = self.weak_transform(inp)
out2 = (out2)
```

We are not yet getting the performance you have in your paper. Could you please elaborate on your experimental setup including hyperparameters? Did you use different transforms? Your help is highly appreciated. 

PS: could you maybe post your used dataloader for NCT? Maybe our version is different to yours.

**Comment by maxwell0027** [comment]
We use the same dataloader as provided in the project. Batch_size 64, num_labeled_sample 16 and lr 1e-3 are recommended. Here are some points which may be helpful to you:
1. make sure the warm-upped model can achieve a satisfactory result.
2. during 2nd stage, transforms used in warmup stage (weak & base) sometimes work better than those default settings (color & weak) in pefat stage. 
3. due to different environment, you can use different seeds to re-run the code.
